.env.example

```diff
@@ -26,4 +26,5 @@ NORTHSTAR_URI=http://northstar.dev:8000
 NORTHSTAR_API_KEY=abc4324
 
 PHOENIX_URI=https://dev.dosomething.org:8888
+PHOENIX_API_URI=https://staging.beta.dosomething.org/api
```

Comment:
@weerd I think that is why i had merge conflicts, but it should be ok because looking at master, the  `PHOENIX_URI` variable is not being used anywhere but the services config, which I update in this PR to be in new `PHOENIX_API_URI` so we should be good there. I will update everything to point to staging though.